packages/atomic/src/components/common/refine-modal/body.ts

```diff
@@ -7,7 +7,7 @@ export const renderRefineModalBody: FunctionalComponentWithChildrenNoProps =
       <aside
         part="content"
         slot="body"
-        class="adjust-for-scroll-bar flex w-full flex-col"
+        class="mr-[calc(-1*(100vw-3rem-100%))] flex w-full flex-col"
```

Comment:
This is fairly complex. Should we keep the explanation comment above the inline style?...